charts/prow/templates/prow_configs.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+data:
+  config.yaml: |-
+    ---
```

Comment:
Given this file is YAML, could we perhaps move some (or all) of the contents of it into values.yaml? Perhaps keep a bare-minimum configuration in here (so that we can use templating to fill in standard parts of the config), and then inject in any additional config provided in values.yaml? The YAML parser should flatten down values as far as I know, making it possible for users to override things specified as part of the defaults too.